tests/evhtests/l7_evh_ingressclass_test.go

```diff
@@ -652,6 +652,11 @@ func TestEVHUpdateIngressClassWithInfraSetting(t *testing.T) {
 		found, _ := objects.SharedAviGraphLister().Get(settingModelName1)
 		return found
 	}, 40*time.Second).Should(gomega.Equal(true))
+	g.Eventually(func() int {
+		_, aviSettingModel := objects.SharedAviGraphLister().Get(settingModelName1)
+		settingNodes1 := aviSettingModel.(*avinodes.AviObjectGraph).GetAviEvhVS()
+		return len(settingNodes1[0].EvhNodes)
+	}, 40*time.Second).Should(gomega.Equal(2))
 	_, aviSettingModel1 := objects.SharedAviGraphLister().Get(settingModelName1)
```

Comment:
@arihantg: Should we take out line 660 and 661?

Reply:
addressed.